packages/buffetjs-icons/src/components/NumberIcon/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const NumberIcon = props => (
```

Comment:
Can we just call it `Number`, to uniformize with other components?

Reply:
Nope Number is a Javascript object.